bitnami/discourse/README.md

```diff
@@ -1,4 +1,6 @@
-# Discourse
+# Discourse&reg;
+
+Trademarks: This software listing is packaged by Bitnami. The respective trademarks mentioned in the offering are owned by the respective companies, and use of them does not imply any affiliation or endorsement.
```

Comment:
Does this need to go before the description?

Reply:
Good catch! Thanks for checking this